moped-editor/src/views/projects/projectView/ProjectComponents/utils.js

```diff
@@ -64,3 +65,26 @@ export function componentFormStateReducer(state, { key, value, action }) {
     return initialComponentFormState;
   }
 }
+
+// See https://github.com/mui/material-ui/issues/10739#issuecomment-1001530270
+export function useAppBarHeight() {
+  const {
+    mixins: { toolbar },
+    breakpoints,
+  } = useTheme();
+  const toolbarDesktopQuery = breakpoints.up("sm");
+  const toolbarLandscapeQuery = `${breakpoints.up(
+    "xs"
+  )} and (orientation: landscape)`;
+  const isDesktop = useMediaQuery(toolbarDesktopQuery);
+  const isLandscape = useMediaQuery(toolbarLandscapeQuery);
+  let currentToolbarMinHeight;
+  if (isDesktop) {
+    currentToolbarMinHeight = toolbar[toolbarDesktopQuery];
+  } else if (isLandscape) {
+    currentToolbarMinHeight = toolbar[toolbarLandscapeQuery];
+  } else {
+    currentToolbarMinHeight = toolbar;
+  }
+  return currentToolbarMinHeight.minHeight;
+}
```

Comment:
😵 - this works really well.

makes me realize we'll want to be able to make the sidebar collapsable in the future. doesn't feel must-have. will make an issue.